### PR TITLE
DNN-8534: Microsoft CodeDOM extension improvements.

### PR DESCRIPTION
**Pull request by zyhfish**

### CODE REVIEW

File: DNN Platform/Components/Microsoft.CodeDom.Providers.DotNetCompilerPlatform/releaseNotes.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿<h3>Url Management</h3>
```
Comment (bdukes) [inline DNN Platform/Components/Microsoft.CodeDom.Providers.DotNetCompilerPlatform/releaseNotes.txt]: Title needs to be updated

File: DNN Platform/Components/Microsoft.CodeDom.Providers.DotNetCompilerPlatform/releaseNotes.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<p class="Contributor">
+	DotNetNuke Corporation <br />
+	<a href="mailto:support@dnncorp.com">support@dnncorp.com</a><br />
+	<a href=http://www.dotnetnuke.com>http://www.dotnetnuke.com</a><br />
+</p>
+<hr/>
+<div class="License">
+	<h3>Microsoft CodeDom .NET Compiler</h3>
+	<p class="Owner">
+		<b>Version CHANGEME</b><br /><br />
```
Comment (bdukes) [inline DNN Platform/Components/Microsoft.CodeDom.Providers.DotNetCompilerPlatform/releaseNotes.txt]: Needs version number

File: DNN Platform/Components/Microsoft.CodeDom.Providers.DotNetCompilerPlatform/releaseNotes.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+	</p>
+	
+	<h2>
+		WARNING - Please check with your hosting provider if they allow having EXE files in the Bin Folder. Installation of this package will result in few EXE files copied under Bin\Roslyn folder. Although, the files are from Microsoft, this will completely break your site if the hosting providers do not allow EXE files in Bin folder.
```
Comment (bdukes) [inline DNN Platform/Components/Microsoft.CodeDom.Providers.DotNetCompilerPlatform/releaseNotes.txt]: :+1:

**Comment by ashishpd** [comment]
@zyhfish can you please make the changes recommended by @bdukes

**Comment by zyhfish** [comment]
changes pushed, thansk @bdukes .

**Pull request merged**